src/main/scala/mesosphere/marathon/api/akkahttp/Rejections.scala

```diff
@@ -14,8 +14,11 @@ object Rejections {
   val defaultEntityNotFoundMessage = Message("Entity was not found")
   case class EntityNotFound(message: Message = defaultEntityNotFoundMessage) extends Rejection
   object EntityNotFound {
-    def app(id: PathId, version: Option[Timestamp] = None): EntityNotFound = {
+    def noApp(id: PathId, version: Option[Timestamp] = None): EntityNotFound = {
       EntityNotFound(Message(s"App '$id' does not exist" + version.fold("")(v => s" in version $v")))
     }
+    def noLeader(): EntityNotFound = {
```

Comment:
Why not `def noLeader: EntityNotFound`? It seems there are no side effects :)

Reply:
I have to admit that I do not like method definitions without the braces much. I'd rather use val.